buildenv/jenkins/JenkinsfileBase

```diff
@@ -181,20 +165,56 @@ def post() {
 					archiveArtifacts artifacts: '**/openjdk-systemtest-results.tar.gz', fingerprint: true, allowEmptyArchive: true
 				}
```

Comment:
- In parallel testing, systemtest result tar file will have same override issue.
Why we need to check test type? Can we just tar the test_output_* dir when the build is unstable?
- Can we prepend $test_target to *.tap file?

Reply:
Updated per request.

* remove check and archive test_output dir if test is unstable.
* since we already archive test_output dir and tap file was in the dir, remove arching tap result step. This will not affect Jenkins tap plugin since tap results were parsed before archiving.